final-project/index.html

```diff
@@ -4,11 +4,251 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./dist/styles.css">
+  <title>Landing Page</title>  
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Quicksand:300,400,500,700|Roboto:400,500" rel="stylesheet"> 
+  <link href="styles/main.css" rel="stylesheet">
 </head>
 <body>
-  <h1>Hello world!</h1>
-  <script src="./dist/bundle.js"></script>
+  <header class="header">
+    <div class="header-elem-1">
+      <span class="logo"></span>
+      <nav class="nav-bar">
+        <a href="">Home</a>
+        <a href="">Blog</a>
+        <a href="">Features</a>
+        <a href="">Pricing</a>
+        <a href="">Documentation</a>
+      </nav>
+    </div>
+    <div class="header-elem-2">
+      <div class="social-icons"> 
+        <div class="fb-logos">
+          <img src="./reference/assets/icons/facebook.png" id="fb-logo">
+          <img src="./reference/assets/icons/facebook_blue.png" id="fb-logo-hover">
+        </div>
+        <div class="tl-logos">
+          <img src="./reference/assets/icons/telegram.png" id="tl-logo">
+          <img src="./reference/assets/icons/telegram_blue.png" id="tl-logo-hover">
+        </div>
+        <div class="tw-logos">
+          <img src="./reference/assets/icons/twitter.png" id="tw-logo">
+          <img src="./reference/assets/icons/twitter_blue.png" id="tw-logo-hover">
+        </div>
+        <div class="md-logos">
+          <img src="./reference/assets/icons/medium.png" id="md-logo">
+          <img src="./reference/assets/icons/medium_blue.png" id="md-logo-hover">
+        </div>        
+      </div>
+      <button>Get Started</button>
+    </div> 
+  </header>
+  
+  <noscript>
+    For full functionality of this page it is necessary to enable JavaScript. 
+    Here are the <a href="http://www.enable-javascript.com" target="_blank"> instructions how to enable JavaScript in your web browser</a>
+  </noscript>
+
+  <main class="main-content">
+    <div class="hero">
```

Comment:
To be more semantically correct, for each of the sections of the page you can use `<section>`